client/tasks/task.tsx

```diff
@@ -25,6 +25,10 @@ export const Task: React.FC< TaskProps > = ( { query } ) => {
 		invalidateResolutionForStoreSelector( 'getTaskLists' );
 	}, [ id ] );
 
+	if ( task && task.container ) {
```

Comment:
Is there another way we could get the tasks and register them through SlotFill?  I'm very hesitant to add back in container rendering here.